### PR TITLE
[FIX]: Register form 

### LINKED ISSUE #240: [FIX]: Register form schema, checkbox and helpertext
## Summary

Fixing Register layout:
- [x] Validation Schema
- [x] Validation Message
- [x] Remove Checkbox

[Figma design](https://www.figma.com/file/McDBByDqFzjYuYZ4CwVpax/UI-v2.0?node-id=3%3A22)

### PR DESCRIPTION
**Pull request by f-morgado**
Relates to #240 

## Screenshots (if visual changes)
image
image

## Proposed Changes

  - User Schema updated
  - Register Form helpertext update and frontend schemas
  - helpertext icon updated
  - emails are all set to lowercase

This pull request closes #240

### CODE REVIEW

File: frontend/src/styles/pages/auth.styles.tsx [hunk 1]
```diff
@@ -1,29 +1,36 @@
-import { styled } from '@stitches/react';
+import { styled } from 'styles/stitches/stitches.config';
 
 import Flex from 'components/Primitives/Flex';
 
 const CenteredContainer = styled('div', {
 	position: 'absolute',
-	top: '150px',
+	top: '50%',
```
Comment (dsousa12) [inline frontend/src/styles/pages/auth.styles.tsx:7]: Please change to `150px`
> This is a change to fix the container position

File: frontend/src/styles/pages/auth.styles.tsx [hunk 2]
```diff
@@ -1,29 +1,36 @@
...
+	backgroundColor: '#ffffff',
 	borderRadius: '$12'
 });
 const ImageBackground = styled(Flex, {
 	height: '100vh',
 	width: '100%',
 	position: 'relative',
 	backgroundColor: '$black',
-	backgroundImage: 'url(/images/background.svg)',
+	backgroundImage: 'url(images/background.svg)',
```
Comment (dsousa12) [inline frontend/src/styles/pages/auth.styles.tsx:26]: Please add `/`before `images`

**Review by dsousa12 — CHANGES_REQUESTED** [review]

**Comment by f-morgado** [comment]
Done, ty for the review

**Review by dsousa12 — APPROVED** [review]
LGTM

**Pull request merged**